tests/storage/test_event_push_actions.py

```diff
@@ -181,6 +186,174 @@ def _mark_read(event_id: str) -> None:
         _rotate()
         _assert_counts(0, 0, 0)
 
+    def test_count_aggregation_threads(self) -> None:
```

Comment:
(I should possibly put this in a docstring...)

This is a "copy" of the `test_count_aggregation` test, but adapted to have both a "main" timeline and a thread in the same room.